src/app/personal-info/personal-info.component.ts

```diff
@@ -150,7 +150,6 @@ export class PersonalInfoComponent implements OnInit {
   }
 
 
-
    /*

```

Comment:
@Bithika-Redapple Be mindful of your formatting. Run your formatting tools so that your code is presentable.

Reply:
okay